Makefile

```diff
@@ -57,4 +57,8 @@ clean-output:
 # standard cleanup target
 clean: clean-output clean-cache
 
-.PHONY: all kind build install clean-cache clean-output clean
+# unit tests (hermetic)
+test:
+	GOOS=linux hack/go_container.sh go test ./...
```

Comment:
TODO: we'll need to add something that dumps the cover profile and invoke gopherage (pinned in hack/tools/go.mod possibly? might even want another module for that as test-infra is a mess to import...)

Reply:
dumping the cover profile now, just need to have CI copy it to artifacts and render it.